backend/src/main/resources/application.yaml

```diff
@@ -74,10 +74,16 @@ server:
   error:
     include-stacktrace: never
 management:
-  endpoint.health.probes.enabled: true
-  endpoint.info.enabled: true
   endpoints.web.exposure.include: health, info
-  endpoint.health.show-components: always
+  endpoint:
+    info:
+      enabled: true
+    health:
+      show-components: always
+      status:
+        http-mapping:
+          okta_degraded: 204
```

Comment:
The pedantic nerd inside me debates the correctness of returning a 204/NO CONTENT.  That's not to say that there is an existing 2xx status code that would be correct.  But I'd almost be tempted to use a 2xx that doesn't have an existing status (like 299 for example).

At the end of the day I don't think it matters.